stories/list-grid/list-grid.stories.js

```diff
@@ -0,0 +1,1880 @@
+                <div tabindex="0" class="fd-grid-list__item fd-grid-list__item--state-draft">
+                    <div class="fd-grid-list__item--body fd-grid-list__item--body--padding">
+                        <div class="grid-list-item-body-example">
+                            <span class="fd-avatar fd-avatar--s grid-list-item-body-example-avatar" role="presentation" aria-label="John Doe"></span>
+                            <div>
+                                <h4 class="fd-title fd-title--h4">John Li</h4>
+                                <p>Product Owner, Company B</p>
+
+                                <div>
+                                    <p>781 Main Street</p>
+                                    <p>Anytown, SD 57401</p>
+                                    <p>USA</p>
+                                </div>
+
+                                <a href="#" class="fd-link" tabindex="0">john_li@example.com</a>
+                            </div>
+                        </div>
+
+                        <div class="fd-grid-list__item--state-actions">
+                            <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Draft">
+                                Draft
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+states.storyName = 'States';
+
+states.parameters = {
+    docs: {
+        storyDescription: `
+To apply state, add the corresponding class \`.fd-grid-list__item--state-(unread|error|locked|draft)\` to the \`.fd-grid-list__item\`.
+    `
+    }
+};
+
+export const highlight = () => `<div class="fd-grid-list">
+    <div class="fd-container">
+        <div class="fd-row">
+            <div class="fd-col fd-col--12">
+                <div class="fd-toolbar fd-toolbar--solid">
+                    <span>Products (420)</span>
+
+                    <span class="fd-toolbar__spacer"></span>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Sort">
+                        <i class="sap-icon--sort"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Filter">
+                        <i class="sap-icon--filter"></i>
+                    </button>
+
+                    <button class="fd-button fd-button--compact fd-button--transparent" aria-label="Group">
+                        <i class="sap-icon--group-2"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-row">
+            <div  class="fd-col fd-col--12 fd-col-md--6 fd-col-lg--4 fd-col-xl--3">
+                <div tabindex="0" class="fd-grid-list__item">
+                    <span class="fd-grid-list__highlight--success"></span>
```

Comment:
There needs to be some screen reader support for the highlight states so screen reader users are aware of a) that it's highlighted at all and b) what the color means. 

This should honestly go back to UX since this is using only color to indicate a state and that is not accessible for several user groups, not just the visually impaired.

Reply:
fixed